lib/lnd_client.ex

```diff
@@ -186,8 +214,10 @@ defmodule LndClient do
     })
   end
 
-  def handle_call({ :connect, details }, _from, state) do
-     case Connectivity.connect(details) do
+  def handle_call(:connect, _from, state) do
+    conn_config = Map.get(state, :conn_config)
+
+    case Connectivity.connect(conn_config |> Map.from_struct) do
```

Comment:
I'd rather `LndClient.Connectivity` to receive a `%ConnConfig{}` rather than doing a `Map.from_struct` from here